partcad/src/partcad/shape_ai.py

```
@@ -8,7 +8,7 @@
 #
 
 import asyncio
-from path import Path
+from pathlib import Path
```

Ruff (0.8.2)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)